frontend/src/features/splReports/columns.tsx

```diff
@@ -53,6 +53,22 @@ export const columns: any[] = [
     width: spacing.medium,
     minWidth: 80,
   },
+  {
+    Header: 'Risk',
+    accessor: 'project.risk',
```

Comment:
Can we change this so that instead of displaying the color name, we display a badge with that color: https://react-bootstrap.github.io/components/badge/

Reply:
This isn't required

Reply:
I think if I add it, that one column would stand out a lot, perhaps too much?